client/packages/programs/src/JsonForms/common/components/Number.tsx

```diff
@@ -14,17 +20,42 @@ import {
 import { Box } from '@mui/system';
 import { FormLabel } from '@mui/material';
 
-export const numberTester = rankWith(3, schemaTypeIs('number'));
+export const numberTester = rankWith(
+  5,
+  or(schemaTypeIs('number'), uiTypeIs('Number'))
+);
+
+type NumberOptions = {
+  /*
+  Options only required if the saved data should be converted to a string
+  representation of the number
+  */
+  output?: 'string';
```

Comment:
I think you can auto detect this from the schema, can't you? (like schemaTypeIs does)

Reply:
Oh yes, good point. Will fix.